### ISSUE TITLE
Database GUI client for management remote desktop

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#4536]
It's a hassle to work with the deployed databases right now.  Working through:
tablet -> remote desktop -> putty -> sudo -> psql
is not great.  Could we convince IMTD to install a database GUI client for us?
tablet -> remote desktop -> GUI client

I've used DBeaver and dbForge but there are [lots of options](https://wiki.postgresql.org/wiki/PostgreSQL_Clients#Windows_GUI_Clients).  If we want to stick with Microsoft there's [a (preview) extension for Azure Data Studio](https://learn.microsoft.com/en-us/sql/azure-data-studio/extensions/postgres-extension) for connecting to Postgres.

Let's submit a Jira ticket to start the conversation.

##  :question:  Spike Question
What GUI database client would IMTD be willing to install for us on our management remote desktops?

Timebox: 1 Jira ticket

**Comment by brindasasi** [comment]
AZ-15692 has been created and waiting on the response

**Comment by brindasasi** [comment]
IMTD team mentioned that this type of request has never been initiated before (!) and they are discussing it internally.

**Comment by brindasasi** [comment]
IMTD team mentioned that if we want any of those tools ,we need to create a new ticket and send it for software assessment

**Comment by brindasasi** [comment]
I'm quite surprised there hasn't been any tools already in use for this. Which tool does the team think would most likely to get approved? What should we ask for ? I can create a new ticket on that. 
@GCTC-NTGC/developers

**Comment by petertgiles** [comment]
Since we're already using a lot of Microsoft tools I bet something from them would be easiest to get approved.  I'm going to try out [Azure Data Studio](https://learn.microsoft.com/en-us/sql/azure-data-studio/quickstart-postgres) with Postgres on my own today to see if it's worth requesting.
